source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -105,7 +105,7 @@ class ProcessorState : public Logger::Loggable<Logger::Id::ext_proc> {
   bool bodyReplaced() const { return body_replaced_; }
   bool partialBodyProcessed() const { return partial_body_processed_; }
 
-  void clearRouteCache(envoy::service::ext_proc::v3::CommonResponse common_response);
+  void clearRouteCache(const envoy::service::ext_proc::v3::CommonResponse& common_response);
```

Comment:
It could be a private function? But let's keep it as it is since there are several other functions like `hasBufferedData`. We could clean them up in a separate PR if needed.